elasticapm/contrib/async_worker.py

```diff
@@ -49,23 +49,20 @@ def main_thread_terminated(self):
                 # add or remove items
                 size = self._queue.qsize()
 
-                six.print_(
-                    "PID %i: ElasticAPM is attempting to send %i pending messages" % (
+                sys.stdout.write(
+                    "PID %i: ElasticAPM is attempting to send %i pending messages\n" % (
                         os.getpid(),
                         size,
                     )
                 )
-                six.print_(
-                    "Waiting up to %s seconds, " % ELASTIC_APM_WAIT_SECONDS,
-                    end=''
-                )
+                sys.stdout.write("Waiting up to %s seconds, " % ELASTIC_APM_WAIT_SECONDS)
```

Comment:
Did you leave out the newline here on purpose?

Reply:
Yes. `six.print_('...', end='')` is equivalent with not writing a newline (same with Python 3's print function).